docs/deep-dive/cli/your-first-cli-plugin.md

```diff
@@ -0,0 +1,54 @@
+---
+id: your-first-cli-plugin
+title: Your first CLI plugin
```

Comment:
I think title/id/sidebar_label should be:
"Creating a CLI Plugin"

Reply:
done